pkg/apis/flowcontrol/types.go

```diff
@@ -259,7 +259,7 @@ type NonResourcePolicyRule struct {
 // FlowSchemaStatus represents the current state of a FlowSchema.
 type FlowSchemaStatus struct {
 	// `conditions` is a list of the current states of FlowSchema.
-	// +listType=associative
+	// +listType=map
```

Comment:
debt from the first model pull. integration is complain the unknown list type, map should be correct form instead i suppose..